src/steth/BaseV2Vault.sol

```diff
@@ -54,6 +60,23 @@ abstract contract BaseV2Vault is sc4626, IFlashLoanRecipient {
 
         priceConverter = _priceConverter;
         swapper = _swapper;
+
+        zeroExSwapWhitelist[_asset] = true;
+    }
+
+    /**
+     * @notice whitelist (or cancel whitelist) a token to be swapped out using zeroExSwap
+     * @param _token The token to whitelist
+     * @param _value whether to whitelist or cancel whitelist
+     */
+    function whiteListOutToken(ERC20 _token, bool _value) external {
+        _onlyAdmin();
+
+        if (address(_token) == address(0)) revert ZeroAddress();
```

Comment:
add test for the "zero address" case

Reply:
[Added](https://github.com/lindy-labs/sandclock-contracts/pull/120/commits/311c30f7c5dde9b770b3c00ae59c0ed55c982870)